apps/cow-fi/styles/global.styles.ts

```
@@ -1,9 +1,10 @@
 'use client'
 
 import { createGlobalStyle } from 'styled-components/macro'
+import { CowProtocolTheme } from 'styled-components'
```

ESLint

[error] 'styled-components' import is restricted from being used. Please import from styled-components/macro.

(no-restricted-imports)